trafilatura/core.py

```diff
@@ -48,7 +48,7 @@ def handle_titles(element, dedupbool, config):
     # children
     else:
         title = deepcopy(element)
-        for child in element.iter():
+        for child in element.getchildren():
```

Comment:
Hi @felipehertzer, I don't see why this change is necessary here, it doesn't affect the extraction of authors.

Reply:
Sorry, I forgot to comment the reason for this change. Some websites like [Moree](https://www.moreechampion.com.au/story/7328821/scamwatch-book-for-your-vaccine-via-a-genuine-website/?src=rss) never finish extracting the data, I did the debugger and found an infinite loop in this iter() , there is only one element but the 'for' never ends. To solve this problem I changed iter() to the getchildren() function and then the process ended correctly.

Reply:
Thanks for the explanation. As a sidenote, maybe this would be necessary for all cases in `core.py` where element children are being iterated on.